src/script/conversation/ConversationRepository.js

```diff
@@ -2595,20 +2594,22 @@ z.conversation.ConversationRepository = class ConversationRepository {
         return this.user_repository
           .get_users_by_id(userIds)
           .then(userEntities => {
-            let actionStringId;
-            let messageStringId;
-            let titleStringId;
+            let actionString;
+            let messageString;
+            let titleString;
 
             const hasMultipleUsers = userEntities.length > 1;
+            const titleSubstitutions = z.util.StringUtil.capitalizeFirstChar(userNames);
+
             if (hasMultipleUsers) {
-              titleStringId = z.string.modalConversationNewDeviceHeadlineMany;
+              titleString = t('modalConversationNewDeviceHeadlineMany', titleSubstitutions);
```

Comment:
The call to `t`with the `titleSubstitution` can be factorized and done later in the code